test/cpp/api/transformer.cpp

```diff
@@ -436,7 +402,178 @@ TEST_F(TransformerTest, TransformerDecoderLayer_gelu) {
 }
 
 TEST_F(TransformerTest, TransformerDecoderLayer_gelu_CUDA) {
-    transformer_decoder_layer_test_helper_gelu(true);
+  transformer_decoder_layer_test_helper_gelu(true);
+}
+
```

Comment:
Is there a reason why we're not flipping the parity for TransformerEncoder on https://github.com/pytorch/pytorch/blob/master/test/cpp_api_parity/parity-tracker.md ?

Reply:
Ah, yes, we will flip the parity. There will be a separate PR to enable all parities later. We are currently focusing on implementation and will resolve parity in later PRs.